apps/server/src/modules/task-card/uc/task-card.uc.ts

```diff
@@ -130,4 +143,13 @@ export class TaskCardUc {
 
 		return taskCard;
 	}
+
+	private replaceCompletionDate(taskCard: TaskCard, newCompletionDate: Date) {
+		taskCard.completionDate = newCompletionDate;
+		if (taskCard.pastCompletionDate()) {
```

Comment:
this means that the (author) decided to change the completion date, e.g. Teacher sais Task-Card should be done by 1st Jan, and then changes his mind and sais it should be done by 1st Feb.
So, the validation pastcompletion - makes sense not when editing the task, but when closing the task - and closing the task is not part of this ticket

Reply:
missunderstanding, will solve validation via decorator